Mobile/stockdogmobile/components/activityPost.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { TouchableOpacity, View, Text } from 'react-native';
+import styles from '../style/screens/feed';
+
+export default class ActivityPost extends Component {
+
+   constructor(props) {
+      super(props);
+   }
+
+   render() {
+      var post = this.props.post;
+      return (
+         <View style={styles.activityPost}>
+            <Text style={styles.miniText}>{post.date}</Text>
+            <View style={styles.postTitleContainer}>
+               <Text style={styles.postTitle}>{post.username}</Text>
+               <Text style={styles.postTitle}>{post.amount} {post.ticker} {post.action}</Text>
+            </View>
+            {
+               post.comments.map((comment, index) => {
```

Comment:
I suggest moving this to a function outside the render function. It will help with both future testing of the functionality and readability of the render function.

Reply:
👍 agreed